src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -75,6 +75,22 @@ public void addConversation(Conversation conversation) {
     persistentStorageAgent.writeThrough(conversation);
   }
 
+  /** Access Conversation by UUID. */
+  public Conversation getConversation(String conversationId) {
```

Comment:
Better practice to instead pass in a UUID instead of converting all UUIDs to strings